k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -110,14 +110,14 @@ function tarPodLogs()
     done
 
     # Get support bundle.  Use first container.
-    local path="/var/cortx/support_bundle"
```

Comment:
Removed leading slash so that later "kubectl cp" does not copy from a path with leading "/".  The slash is added in the few places where needed.